mint.json

```diff
@@ -146,6 +146,15 @@
       "group": "SDKs",
       "pages": ["integrations/sdks/speakeasy", "integrations/sdks/stainless"]
     },
+    {
+      "group": "User Auth",
```

Comment:
I've been wondering - is Auth the correct term for this? And do we want to specify authentication? cuz auth is ambiguous and can be authentication or authorization

Reply:
imo auth is fine to use here. it's usually safe to assume that auth means both authentication and authorization

Reply:
I feel like auth is fine here because it's doing a bit of both. When we do hidden pages, there will be an element of authentication (first verify the user is who they say they are), then an element of authorization (tell us what pages the user should have access to). I don't want to restrict it to one or the other, but if I had to say one I'd definitely say authentication because that is always a prerequisite for authorization - you can't tell someone what they can/can't do (authorization) until you know who they are (authentication)